src/functions/max.js

```diff
@@ -1,20 +1,44 @@
 import {constants} from "../core/constants";
 import {wrap} from "../core/wrap";
 
-// Get the maximum value in a sequence as judged by a relational function.
-// If no relational function is provided, then (a, b) => (a < b) is used.
+import {NotBoundedError} from "../errors/NotBoundedError";
+
 export const max = wrap({
     name: "max",
+    summary: "Get the maximum element in a sequence.",
+    docs: process.env.NODE_ENV !== "development" ? undefined : {
+        expects: (`
+            The function expects a sequence as input an an optional
+            [relational function].
+            If no relational function was passed, the default relational
+            function \`(a, b) => (a < b)\` is used.
+        `),
+        returns: (`
+            The function returns the maximum element in the sequence according
+            to the relational function. Functionally, this would be the same
+            as [stable-sorting](stable sort) the sequence according to that
```

Comment:
Created an issue for this so it isn't forgotten -- I would totally forget to do this.
https://github.com/pineapplemachine/higher/pull/66/files#r127577462